Documentation/security/policy/language.rst

```diff
@@ -1562,6 +1562,27 @@ the label ``type=ingress-worker`` on TCP ports 22, 6443 (kube-apiserver), 2379
 To reuse this policy, replace the ``port:`` values with ports used in your
 environment.
 
+In order to allow protocols such as VRRP and IGMP that don't have any transport-layer
+ports, set ``--enable-extended-ip-protocols``` flag to true. By default, such traffic is
+dropped with ``DROP_CT_UNKNOWN_PROTO``` error.
+
+As an example, the following policy allows egress traffic on any node with
+the label ``type=egress-worker`` on TCP ports 22, 6443/443 (kube-apiserver), 2379
+(etcd), and 4240 (health checks), UDP port 8472 (VXLAN), and traffic with VRRP protocol.
+
+.. only:: html
+
+   .. tabs::
+     .. group-tab:: k8s YAML
+
+        .. literalinclude:: ../../../examples/policies/host/allow-extended-protocols.yaml
+          :language: yaml
+
+.. only:: epub or latex
+
+        .. literalinclude:: ../../../examples/policies/host/allow-extended-protocols.yaml
+          :language: yaml
```

Comment:
Please take a look at https://github.com/cilium/cilium/pull/40212/files#diff-a0b27a8b1488de6bbbb7e5f8282627bcff6d32826f2aa2341d8af07387df34b7, it's probably worth ditching the different format conditions here to align with Casey's clean-up.

Reply:
https://github.com/cilium/cilium/pull/40212 has just be merged, please rebase.